src/backend/db/products.js

```diff
@@ -7,6 +7,7 @@ import { v4 as uuid } from "uuid";
 
 export const products = [
   {
+    id:1,
```

Comment:
why create two ids here? There is already a _id is available.

Reply:
One of the IDs is taking value from uuid() .. so it will change everytime, for the timebeing i have added one more id so that it would be easier when I will create individual pages for each product ..